deno_typescript/compiler_main.js

```diff
@@ -163,18 +164,17 @@ class Host {
         .replace("/index.d.ts", "");
     }
 
-    let { sourceCode, moduleName } = dispatch("readFile", {
+    const { sourceCode, moduleName } = dispatch("readFile", {
       fileName,
       languageVersion,
       shouldCreateNewSourceFile
     });
 
-    // TODO(ry) A terrible hack. Please remove ASAP.
-    if (fileName.endsWith("typescript.d.ts")) {
-      sourceCode = sourceCode.replace("export = ts;", "");
-    }
```

Comment:
Was it not necessary? How'd you get this to work? : )

Reply:
By augmenting the global type scope... `typescript.d.ts` is a "modular"/UMD `.d.ts` because it contains an import/export.  This was removing the on export, which changed it to a global/ambient `.d.ts`, which was a hacky way to deal with it as below.